test/functionalTest/cases/2670_milliseconds_for_DateTime_attrs/2670_milliseconds_for_DateTime_attrs.test

```diff
@@ -0,0 +1,90 @@
+# Copyright 2020 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Milliseconds for a DateTime attribute
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0
+
+--SHELL--
+
+#
+# 01. Create an Entity E1, with an attribute 'ts' that is of DateTime type and with a value with milliseconds
+# 02. GET E1 and see its 'ts' attribute with its milliseconds
+#
+
+echo "01. Create an Entity E1, with an attribute 'ts' that is of DateTime type and with a value with milliseconds"
+echo "==========================================================================================================="
+payload='{
+  "id": "E1",
+  "ts": {
+    "type": "DateTime",
+    "value": "2020-07-03T09:24:04.456"
+  }
+}'
+orionCurl --url /v2/entities --payload "$payload"
+echo
+echo
+
+
+echo "02. GET E1 and see its 'ts' attribute with its milliseconds"
+echo "==========================================================="
+orionCurl --url /v2/entities/E1
+echo
+echo
+
+
+--REGEXPECT--
+01. Create an Entity E1, with an attribute 'ts' that is of DateTime type and with a value with milliseconds
+===========================================================================================================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/entities/E1?type=Thing
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+02. GET E1 and see its 'ts' attribute with its milliseconds
```

Comment:
Maybe we should test also the value is correctly rendered in notifications?

Reply:
There are many many functests already with rendered milliseconds.

Reply:
What I mean, if at the end a new .test (2670_milliseconds_for_DateTime_attrs.test) has been added to check milliseconds rendering (which I think is a good idea) let's do it checking the two main ways of rendering context information: synchronous GET and asynchronous notifications.

Reply:
Sure, no probs

Reply:
Fixed in 6d18bc1